common/domain-setter.sub.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>A page that will likely be same-origin-domain but not same-origin</title>
+
+<script>
+"use strict";
+document.domain = "{{host}}";
```

Comment:
Can you give an example? I thought that the "If the given value is not a registrable domain suffix of and is not equal to effectiveDomain" check would prevent e.g. setting `document.domain = "example.com"`.